### ISSUE TITLE
Discussion: more Defacto standard default release tagging approach.

### ISSUE THREAD

**Issue by jandersson-svt** [allegro/axion-release-plugin#367]
Proposal: Axion should provide a more semver friendly tagging out of the box. Instead of using the  prefix/separator concept as default, that tags the releases as  nonsemver "release-X.Y.Z" I propose that the prefix concept is changed to a postfix/build concept, or according to semver: https://semver.org/
` <version core> "+" <build> `
i.e so instead of the out-of-the box release tag
 release-X.Y.Z
the default release tag would be
X.Y.Z-release

OR/AND possible the non-semver historically often used variant 
vX.Y.Z

I.e. as default v instead of release-.

Besides acting even more as a good semver citizen, this would also make the incompatibly that is today if you change prefix for tags during a project phase go away. Example today
: start release tagging as default
: decides to change prefix to something else, like non prefix '' and non separator '' for generating de-facto tag X.Y.Z
-> axion crasches on old release-x.y.z tags being nonsemver. delete or rename old default releaste-tags.

And it would still work with the axion multiproject concept with different versions (which does not work without an release/prefix identifier).

I had a look at the source code and I understand this change would be some work, not supermuch, but not supersmall either, as the prefix-concept is a bit of an embedded concept. However, I think I would be worth it, and if the idea is liked, I could start working on a PR for such a change, If you the maintainer does not have the time for it.

TL;DR:
Remove the property prefix, replace with postfix/build property - in the name of Semver + and current strategy name change issues. OR replace the default release- with v.

**Comment by jandersson-svt** [comment]
I left a little PR that would change the default prefix to v instead of release-, and also would fulfill this discission proposals 

https://github.com/allegro/axion-release-plugin/pull/377

**Comment by jandersson-svt** [comment]
After #377 was merged, sometime in the future I would like to inspect the possibility to allow pure semVer tags, (X.Y.Z-asdf) - which is possible today, but If i recall correct, there was an issue with mulitprojects currently if doing so - However, I think that would be needed to be investigated in a separate and focused issue/pr. I feel this discussion issue is a bit vague after #377  and I would close this discussion if fine with you @bgalek  :)

**Comment by bgalek** [comment]
@jandersson-svt hi! sure, pure semVer seems a reasonable way to go. I'll need some time to dive into multiproject issues, hope we will agree on something one day ;)

BTW.
I'm just working to releasing new version ;)

**Comment by jandersson-svt** [comment]
After discussion and partial solution patched, closing this.
